Bottom: f1580d9cca758a2c236b908387608769f225ecb3
Top:    a2dc04c0382557b03c921d5b0e8b6f853c614274
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-28 01:01:10 +0100

Refresh of temporary-comment-out-some-cal

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 5e70165..fb496f0 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1956,9 +1956,9 @@ PanAction (int argc, char **argv, int x, int y)
     }
   else if (x == on_x && y == on_y)
     {
-      ghid_show_crosshair (FALSE);
+      //ghid_show_crosshair (FALSE);
       ghidgui->auto_pan_on = !ghidgui->auto_pan_on;
-      ghid_show_crosshair (TRUE);
+      //ghid_show_crosshair (TRUE);
     }
 
   return 0;
